storybookjs/storybook
# Fixing withBackgrounds on config.js

Issue:

## What I did
Fixed the example of how add backgrounds on all stories

## How to test
Try import withBackgrounds as an object

Is this testable with Jest or Chromatic screenshots? No
Does this need a new example in the kitchen sink apps? No
Does this need an update to the documentation? Yes

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`
